lib/fork.js

```diff
@@ -153,7 +153,14 @@ module.exports = function (file, opts, execArgv) {
 		return promise;
 	};
 
+	var isExitInitiated = false;
```

Comment:
I removed this completely and tests ran successfully, I guess there's no need for it.